batteryml/preprocess/preprocess_RWTH.py

```diff
@@ -0,0 +1,195 @@
+# Licensed under the MIT License.
+# Copyright (c) Microsoft Corporation.
+
+import shutil
+from typing import List
+import zipfile
+import numpy as np
+import pandas as pd
+
+from tqdm import tqdm
+from numba import njit
+from pathlib import Path
+
+from batteryml import CycleData, BatteryData, CyclingProtocol
+from batteryml.builders import PREPROCESSORS
+from batteryml.preprocess.base import BasePreprocessor
+
+
+@PREPROCESSORS.register()
+class RWTHPreprocessor(BasePreprocessor):
+    def process(self, raw_file) -> List[BatteryData]:
+        raw_file = Path(raw_file)
+
+        # Unzip the file first
+        print('[INFO] Unzip the RWTH raw.zip file. There are'
+              ' many levels of compression in this dataset.',
+              flush=True)
+        subdir = raw_file.parent / 'RWTH-2021-04545_818642'
+        if not (subdir / 'Rawdata.zip').exists():
```

Comment:
When I download RWTH dataset to ./data/raw/RWTH, I got RWTH.zip as defined in download.py, not Rawdata.zip. and The name of the zip file needs to be the same in download.py and here.